static/src/javascripts/es6/projects/common/modules/crosswords/series.js

```diff
@@ -0,0 +1,28 @@
+/*eslint-disable no-new*/
+
+import qwery from 'qwery';
+import config from 'common/utils/config';
+import proximityLoader from 'common/utils/proximity-loader';
+import Series from 'common/modules/onward/onward-content';
+
+function insertOrProximity(selector, insert) {
+    if (window.location.href) {
```

Comment:
@rich-nguyen 
I just duplicated the functionality used on other pages here: https://github.com/guardian/frontend/blob/cbfcd4adeda892b10b0ed970878ec4b67db96104/static/src/javascripts/bootstraps/trail.js#L43 
Maybe that's over abstraction and I don't need this